core/tests/protractor_desktop/collections.js

```diff
@@ -40,22 +38,17 @@ describe('Collections', function() {
   var libraryPage = null;
 
   beforeAll(async function() {
-    adminPage = new AdminPage.AdminPage();
     creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
     collectionEditorPage = new CollectionEditorPage.CollectionEditorPage();
     libraryPage = new LibraryPage.LibraryPage();
     var EDITOR_USERNAME = 'aliceCollections';
     var PLAYER_USERNAME = 'playerCollections';
     var CREATOR_USERNAME = 'creatorExplorations';
     await users.createUser('creator@explorations.com', CREATOR_USERNAME);
-    await users.createUser('player@collections.com', PLAYER_USERNAME);
-    await users.createUser('alice@collections.com', EDITOR_USERNAME);
-    await users.createAndLoginCurriculumAdminUser(
-      'testadm@collections.com', 'testadm');
-    await adminPage.get();
-    await adminPage.addRole(EDITOR_USERNAME, 'collection editor');
-    await adminPage.addRole(PLAYER_USERNAME, 'collection editor');
-    await users.logout();
+    await users.createCollectionEditor(
+      'player@collections.com', PLAYER_USERNAME);
```

Comment:
Why are we adding a "player" as a collection editor? I'm not sure that makes sense...

Reply:
We have this as of now (see line 57 in old file), I tried changing the email but I realised we have other collection editor below as well.